### PR TITLE
Test summary pages links

### PR DESCRIPTION
**Pull request by gshefer**
Test summary pages links
{{pytest: cfme/tests/containers/test_summary_pages_links.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 330fc4f08d8da90dfd374c8b3c3dc5584de48733:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_summary_pages_links.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+DataSet = namedtuple('DataSet', ['object', 'breadcrumb_member'])
+
+DATA_SETS = [DataSet(Service, 'Container Services'),
+             DataSet(Route, 'Routes'),
+             DataSet(Project, 'Projects'),
+             DataSet(ContainersProvider, 'Containers Providers')]
+
+
+@pytest.mark.parametrize(('cls'), DATA_SETS)
+def test_summary_pages_links(privider, cls):
```
Comment (psav) [inline cfme/tests/containers/test_summary_pages_links.py]: Spelling mistake here causes tests to fail ```privider``` -> ```provider```

**Comment by dajoRH** [comment]
Lint report for commit e5c868cb798f89eb609ce46f095615ad91dea39b:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_summary_pages_links.py (hunk 1 above)
Comment (gshefer) [inline cfme/tests/containers/test_summary_pages_links.py]: Done.

**Comment by dajoRH** [comment]
Lint report for commit 292a068cc34567fe1350f024820af4322594fb79:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 312b214b197709f8c5cba507259a14775ebe3ae5:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 600f72c0ed1c07a368ca234a52483e3c856c16fa:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_summary_pages_links.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+DataSet = namedtuple('DataSet', ['object', 'breadcrumb_member'])
+
+DATA_SETS = [DataSet(Service, 'Container Services'),
+             DataSet(Route, 'Routes'),
+             DataSet(Project, 'Projects'),
+             DataSet(ContainersProvider, 'Containers Providers')]
+
+
+@pytest.mark.parametrize(('cls'), DATA_SETS)
```
Comment (psav) [inline cfme/tests/containers/test_summary_pages_links.py]:
```python
@pytest.mark.parametrize(('cls'), DATA_SETS, ids=[cls.object.__name__ for cls in DATA_SETS])
```
Will give you much nicer test names

File: cfme/tests/containers/test_summary_pages_links.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    bread_names = map(sel.text_sane, breads)
+
+    assert cls.breadcrumb_member in bread_names
+
+    chosen_link = filter(lambda b: sel.text_sane(b) ==
+                         cls.breadcrumb_member, breads)[0]
+
+    sel.click(chosen_link)
+
+    # TODO: replace with normal 'am_i_here' function when available
```
Comment (psav) [inline cfme/tests/containers/test_summary_pages_links.py]: This will actually be replaced with a widgetastic view.is_displayed

**Comment by dajoRH** [comment]
Lint report for commit a5b27ceb0b13502433cfbd05b24819a1f602a7f3:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f0129526636d9a5d279eb7b54915224f9c83e12f:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ccb042cfaa53be20e0af0380c01c046ca611a6ac:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7cf3c8d26197947c913d0696f4ef5efd94b3b296:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4c6da495c678683be03c280971b2df8855e29752:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 283df438951fcc090e619cdad1658e19a907d8e8:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b27935c0ff65dc80402a7cdfce8826a8d6c92104:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 665041ef0091954992373fff266a48182759191d:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3eb88777fef6009a2be3e2eccf17deb784ee0edb:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 599048328db6580bcde56ad763169477f84056b0:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**